Microsoft.Toolkit.Uwp.SampleApp/Pages/About.xaml

```diff
@@ -4,6 +4,7 @@
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
       xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:primitiveControls="using:Microsoft.Toolkit.Uwp.UI.Controls.Primitives"
```

Comment:
We should still keep all the controls in the `Microsoft.Toolkit.Uwp.UI.Controls` namespace I think, we're just going to split them into the separate packages. Otherwise, the uber package doesn't help anything as you still need to know where all the controls live.

This I think addresses your other question about the ones already in the Primitives namespace. Though maybe we need a different name that `Primitives` for the package?

Reply:
Put everything back in `Microsoft.Toolkit.Uwp.UI.Controls` b6bc173